can_tools/scrapers/official/federal/HHS/facility.py

```diff
@@ -105,6 +103,15 @@ def normalize(self, data: str) -> pd.DataFrame:
         temp = temp + df.eval(f"{pps} / {ppc}").fillna(0.0)
         df.loc[:, "inpatient_beds_used_covid_7_day_canavg"] = temp.values
 
+        # Combine adult and pediatric covid admissions to find total.
+        # NOTE: Theoretically, this could be missing data from the weekly total
+        # if hospitals do not report everyday, but this seems to be the methodology that
+        # the CDC is following for weekly hospital admissions
```

Comment:
For my own understanding, how can you tell what methodology CDC is using?

I think in practice it doesn't matter because looking at the data, 95% of hospitals have _coverage of 7 and only 0.5% of hospitals have _coverage<7 and _admissions>0 (in the past week anyway, might be different during a surge).  But based on my understanding of these columns, the most correct data would be `(_sum/_coverage)*7` to get an approximated 7-day total.  But we're literally talking about 26 facilities and most of them have coverage of 6, so it wouldn't change the numbers that much anyway.

Can you adjust the NOTE to be clearer about our rationale here?  And if the rationale is "We think the CDC is using the raw _sum and only 0.5% of hospitals have _coverage<7 and _sum>0 so it doesn't matter much." that would be fine.